src/app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks.ts

```diff
@@ -148,9 +148,12 @@ export function useUpdateLedgerSpecificNativeSegwitUtxoHexForAdddressIndexZero()
       )
     );
     inputSigningConfig.forEach(({ index }) => {
-      tx.updateInput(index, {
-        nonWitnessUtxo: Buffer.from(inputsTxHex[index], 'hex'),
-      });
+      // don't update nonWitnessUtxo if it already exists
```

Comment:
@kyranjamie - we discussed doing this but as it doesn't actually fix the bug with multi-sig maybe I am better off not adding it?

Reply:
I think we should add, this is how it works on other wallets. Can you add analytics events to these actions as well 🙏🏼

Reply:
Sure, I can add 👍